helix-view/src/editor.rs

```diff
@@ -449,6 +448,12 @@ pub struct Editor {
 
     pub syn_loader: Arc<syntax::Loader>,
     pub theme_loader: Arc<theme::Loader>,
+    /// last_theme is used for theme previews. We store the current theme here,
+    /// and if previewing is cancelled, we can return to it.
+    pub last_theme: Option<Box<Theme>>,
```

Comment:
Sorry about this, since editor is only initialized once, I don't think we need to `Box` this, can just use `Option<Theme>`.